src/with-urql-client.tsx

```diff
@@ -72,6 +72,12 @@ function withUrqlClient<T = any, IP = any>(
       );
     };
 
+    // Set the correct displayName in development.
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
I'm not sure if we want to limit this to production, the react-devtools are often used in combination with prod environments and this would be a discrepancy in what they see. I think if people want to get rid of things like this it's better to be in a babel-plugin.

Reply:
Ah ok, so you're thinking we should just always leave it in and it's up to the end user to remove it if they see fit? I'm good with that, it keeps behavior in our lib more consistent 👍

Reply:
Yes, exactly!